Ginger/Ginger/BusinessFlowPages_New/ActionsListViewPage.xaml.cs

```diff
@@ -133,12 +134,15 @@ private void SetListView()
             mActionsListView.ItemDropped += listActions_ItemDropped;
 
             mActionsListView.List.MouseDoubleClick += ActionsListView_MouseDoubleClick;
+
+            // Enable Virtualization for Actions ListView to improve the loading time/performance
```

Comment:
No need to set the "CanContentScroll" as true because it is True as defualt, only in case of False like in Acivities which should set it from code